subscriptions/management/commands/add_prepend_next_to_subscriptions.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("A command to add subscription metadata in order to "
+            " inform subscribers of new or changed features.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--audio-file',
+            dest="audio_file",
+            type=str,
+            help='A path to the audio file containing the "\
+            "notification message'
+        )
+
+    def handle(self, *args, **options):
+        audio_file = options['audio_file']
+        if not audio_file:
+            self.warning("audio-file is required.")
+            return
+
+        active_subscriptions = Subscription.objects.filter(
+            active=True,
+            messageset__content_type="audio")
+
+        count = 0
+        for active_subscription in active_subscriptions.iterator():
+
+            if (active_subscription.metadata.get("prepend_next_delivery")
+                    is None):
+                active_subscription.metadata["prepend_next_delivery"] = \
+                    audio_file.replace('<LANG>', active_subscription.lang)
```

Comment:
What about using the python syntax here, eg `{{lang}}`, that way we can just do `audio_file.format(**model_to_dict(active_subscription))`, and you can use any part of the model, not just the lang (for future). 

`model_to_dict` is found in `django.forms.models`